numismatic/feeds/cryptocompare.py

```diff
@@ -65,13 +84,39 @@ def get_historical_minute(self, fsym, tsym, e=None, limit=30, toTs=None):
         params = dict(fsym=fsym, tsym=tsym, e=e, limit=limit, toTs=toTs)
         return self._make_request(api_url, params)
 
-    def _make_request(self, api_url, params=None):
-        data = super()._make_request(api_url, params)
-        if 'Data' in data:
+    def _make_request(self, api_url, params=None, raw=False):
+        data = super()._make_request(api_url, params, raw=raw)
+        if 'Data' in data and not raw:
             data = data['Data']
         return data
 
+    @staticmethod
+    def parse_price(msg):
```

Comment:
Can you please put the format of `msg` in the comments like I do for Poloniex. It makes it MUCH easier to understand the code and for a third party to debug/add.

Reply:
I must look at that in the poloniex code. Didn't notice it. Please can you open an issue for this?